artifactory.py

```diff
@@ -245,6 +245,25 @@ def chunks(data, size):
         yield {k: data[k] for k in islice(it, size)}
 
 
+def print_download_progress(bytes_now, total_size):
```

Comment:
Sorry that I'm an asshole, but now it's `log_progress` :) 
We can also remove a docstring, imho

Reply:
now that makes sense, done!